### PR TITLE
edited notes

### PR DESCRIPTION
**Pull request by GrantHiggins16**
ya boi can do `git`

### CODE REVIEW

**Comment by ejmg** [comment]
-1

**Comment by ejmg** [comment]
@nucle0tides your take? pass code review?

**Comment by nucle0tides** [comment]
For the sake of consistency, I recommend using past tense for commit messages, but for future projects I recommend [present tense](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) for commit messages.

**Comment by nucle0tides** [comment]
+1 for consistency, -1 because tough love gives you a thick skin.

File: notes.org [hunk 1]
```diff
@@ -28,6 +28,8 @@
 *** Filings
 **** 10K
      - annually required filing by SEC that gives comprehensive report on a company's performance
+     - balance sheets seem to be kept in <B> tag as either Balance sheets or Consolidated Balance sheets
```
Comment (nucle0tides) [inline notes.org]: you're a computer scientist. computer scientists use the 'heuristic' any and every chance they get. i recommend changing 'balance sheets seem to be kept in...' to 'heuristic analysis suggests balance sheets are kept in...'

Comment (nucle0tides) [inline notes.org]: so heuristic analysis is apparently an anti-virus thing. @ feds don't arrest me.

**Pull request merged**